config/mvc-address-lookup.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+return [
+    'default' => env('ADDRESS_LOOKUP_SERICE', 'PdokService'),
+    'providers' => [
```

Comment:
als je provider nog geen config verwacht zoals tokens of urls etc. dan mag dit weg.

Ik bedenk me nu wel dat we hier requests doen naar een url, dus ik zou die base url hiernaar toe verplaatsen. Als pdok dan een x ze url veranderd kunnen we dat via de config aanpassen inplaats van in de code.